pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbouncer_types.go

```diff
@@ -123,6 +127,13 @@ type PGBouncerPodSpec struct {
 	TopologySpreadConstraints []corev1.TopologySpreadConstraint `json:"topologySpreadConstraints,omitempty"`
 }
 
+// PGBouncerSidecars defines pgBouncer sidecar containers
+type PGBouncerSidecars struct {
+	// Defines the configuration for the pgBouncer config sidecar container
+	// +optional
+	PGBouncerConfig *Sidecar `json:"pgbouncerconfig,omitempty"`
```

Comment:
:wrench: Thinking we should camelCase here (e.g. `pgbouncerConfig`) for consistency with other areas of the spec.

Reply:
:+1: updated.